src/components/Chat/Chat.tsx

```diff
@@ -20,7 +21,7 @@ enum AskOptions {
 }
 const ASK_OPTIONS = Object.values(AskOptions);
 
-const PROMPT_OPTIONS = ["Generate weekly 1-1 talking points from this file"]; // more options to come
+const PROMPT_OPTIONS = ["Generate weekly 1-1 talking points from this file", "Separate concepts from todos"]; // more options to come
```

Comment:
Perhaps cursor-pointer would be nice over the prompt suggestion buttons

Reply:
I think it already has cursor pointer though?

Reply:
<img width="304" alt="Screenshot 2024-03-03 at 23 44 03" src="https://github.com/reorproject/reor/assets/17236551/3474c611-3d7f-4019-9f12-f459eba48dc2">
hovering here ^ it doesn't